layout/wrapper.html

```diff
@@ -87,7 +87,10 @@
 			<div class="n-layout__row n-layout__row--footer">
 				{{#outputBlock 'footer'}}{{/outputBlock}}
 				{{#if @root.flags.welcomePanel}}
-					{{>n-ui/welcome-message/template}}
+					{{#ifEquals nUi.welcomeMessage false}}
```

Comment:
Unfortunately not, as that would mean updating every app that uses this template (as `undefined` would count as a positive)